src/scenes/AcceleratorRouter/Cohorts/CohortListView.tsx

```diff
@@ -25,6 +25,11 @@ const columns = (activeLocale: string | null): TableColumnType[] =>
           name: strings.PHASE,
           type: 'string',
         },
+        {
```

Comment:
The `columns` passed in from this component is unused. We could remove `columns` from here.

Reply:
Oh it's pretty obvious too 😅 Thanks for catching 🙌